definitions/ext-istio_service/definition.yml

```diff
@@ -22,6 +22,88 @@ synthesis:
+      prefix: istio_request
+    tags:
+      service_istio_io_canonical_name:
+        entityTagNames: [label.service.istio.io/canonical-name, service_istio_io_canonical_name]
+      service_istio_io_canonical_revision:
+        entityTagNames: [label.service.istio.io/canonical-revision, service_istio_io_canonical_revision]
+      app:
+        entityTagNames: [label.app, app]
+      version:
+        entityTagNames: [label.version, version]
+      label.istio:
+        entityTagNames: [label.istio, istio]
+      istio_io_rev:
+        entityTagNames: [label.istio.io/rev, istio_io_rev]
+      operator_istio_io_component:
+        entityTagNames: [label.operator.istio.io/component, operator_istio_io_component]
+      k8s_cluster_name:
+        entityTagNames: [k8s.cluster.name, k8s_cluster_name]
+      clusterName:
+      namespaceName:
+      deploymentName:
+      
+  - identifier: "service_istio_io_canonical_name"
+    encodeIdentifierInGUID: true
+    name: "service_istio_io_canonical_name"
+    conditions:
+    - attribute: metricName
+      prefix: istio_response
+    tags:
+      service_istio_io_canonical_name:
+        entityTagNames: [label.service.istio.io/canonical-name, service_istio_io_canonical_name]
+      service_istio_io_canonical_revision:
+        entityTagNames: [label.service.istio.io/canonical-revision, service_istio_io_canonical_revision]
+      app:
+        entityTagNames: [label.app, app]
+      version:
+        entityTagNames: [label.version, version]
+      label.istio:
+        entityTagNames: [label.istio, istio]
+      istio_io_rev:
+        entityTagNames: [label.istio.io/rev, istio_io_rev]
+      operator_istio_io_component:
+        entityTagNames: [label.operator.istio.io/component, operator_istio_io_component]
+      k8s_cluster_name:
+        entityTagNames: [k8s.cluster.name, k8s_cluster_name]
+      clusterName:
+      namespaceName:
+      deploymentName:
+
+  - identifier: "service_istio_io_canonical_name"
+    encodeIdentifierInGUID: true
+    name: "service_istio_io_canonical_name"
+    conditions:
+    - attribute: metricName
+      prefix: istio_tcp
+    tags:
+      service_istio_io_canonical_name:
+        entityTagNames: [label.service.istio.io/canonical-name, service_istio_io_canonical_name]
+      service_istio_io_canonical_revision:
+        entityTagNames: [label.service.istio.io/canonical-revision, service_istio_io_canonical_revision]
+      app:
+        entityTagNames: [label.app, app]
+      version:
+        entityTagNames: [label.version, version]
+      label.istio:
+        entityTagNames: [label.istio, istio]
+      istio_io_rev:
+        entityTagNames: [label.istio.io/rev, istio_io_rev]
+      operator_istio_io_component:
+        entityTagNames: [label.operator.istio.io/component, operator_istio_io_component]
+      k8s_cluster_name:
+        entityTagNames: [k8s.cluster.name, k8s_cluster_name]
+      clusterName:
+      namespaceName:
+      deploymentName:
 
   tags:
```

Comment:
Note to me: check if these will be tried for the rules that have specific tags definitions

Reply:
According to docs the specific tags take precedence.